jdaviz/configs/cubeviz/plugins/slice/slice.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <j-tray-plugin
-    description='Select slice (or wavelength) of the cube to show in the image viewers and highlighted in the spectrum viewer.  The slice can also be changed interactively in the spectrum viewer by activating the slice tool.'
+    :description="docs_description || 'Select slice of the cube to show in the image viewers.  The slice can also be changed interactively in the spectrum viewer by activating the slice tool.'"
```

Comment:
This doesn't seem related?

Reply:
It is, it's covered in the bullet "allowing overriding the plugin header text when inherited by downstream apps" and coupled with setting `docs_description` as an empty traitlet in the plugin template mixin.

Reply:
would you rather I split this into a separate PR (and then add the hookups for all the plugins there)?

Reply:
moved (and applied to all plugins) in https://github.com/spacetelescope/jdaviz/pull/2709